hazelcast/src/main/resources/hazelcast-config-4.0.xsd

```diff
@@ -3661,6 +3661,222 @@
         </xs:attribute>
     </xs:complexType>
 
+    <xs:element name="secure-store-substitution-group" abstract="true"/>
+
+    <xs:complexType name="keystore-entry">
+        <xs:annotation>
+            <xs:documentation>
+                The key entry stored in a Java KeyStore.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:attribute name="name" type="xs:string" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    The name of the entry.
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="password" type="xs:string" use="required">
```

Comment:
Usually, entry passwords are optional. When it's not provided, the password from the keystore itself is used.

Reply:
+1

Reply:
Entries are gone now.